modules/persistent-storage-csi-smb-cifs-create-sc.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-smb-cifs.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-smb-cifs-create-sc_{context}"]
+= Creating a storage class for CIFS/SMB
+
+After installing the Operator, you should create a storage class for dynamic provisioning of Common Internet File System (CIFS) dialect/Server Message Block (SMB) protocol volumes.
+
+.Prerequisites
+* You are logged in to the running {product-title} cluster.
+
+* You have SMB server installed and know its:
+** hostname
+** share name
+** username/password
```

Comment:
Unless these are specific terms that need to be lowercase, I think these should be capitalized:
```suggestion
** Hostname
** Share name
** Username/password
```